src/components/elements/FilterChips.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import Chip from 'material-ui/Chip'
+import ByPath from 'object-path'
+
+export const FilterChips = ({filter, selectSpecificFilter, records, onRequestDelete}) => {
+    let selectedFilters = ByPath.get(filter, selectSpecificFilter)
+    return selectedFilters? selectedFilters.map((selectedFilter) => hydrateChip(selectedFilter, records))
+        .map(chip => renderChip(chip.id, chip.name, onRequestDelete(filter, chip.id)))
+        :
+        <div></div>
+}
+
+export const mapStateToProps = (state, ownProps) => {
```

Comment:
This there's no body here, this can use the same format as hydrateChip for just returning an object.